setup.py

```diff
@@ -432,6 +438,7 @@ def get_sphinx_theme_version() -> str:
 singularity = ['spython>=0.0.56']
 slack = [
     'slack_sdk>=3.0.0,<4.0.0',
+    'requests>=2.20.0',
```

Comment:
I couldn't see anywhere that `requests` is used in the slack provider
```suggestion
```

Reply:
SlackWebHook derives from SimpleHttpHook

Reply:
Shouldn't that just be a cross-dep to the http provider then?

Reply:
But yeah, SlackWebHook is only usable when optional [http] extra on slack provider, which pulls in requests anyway. So indeed - we can remove it from here.